pkg/ec2pricing/ec2pricing.go

```diff
@@ -1,355 +1,115 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
 package ec2pricing
 
 import (
-	"fmt"
-	"math"
-	"sort"
-	"strconv"
-	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws/endpoints"
-	"go.uber.org/multierr"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
-	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
 	"github.com/aws/aws-sdk-go/service/pricing"
-	"github.com/aws/aws-sdk-go/service/pricing/pricingiface"
+	"go.uber.org/multierr"
 )
 
 const (
-	defaultSpotDaysBack = 30
-	productDescription  = "Linux/UNIX (Amazon VPC)"
-	serviceCode         = "AmazonEC2"
+	productDescription = "Linux/UNIX (Amazon VPC)"
+	serviceCode        = "AmazonEC2"
+)
+
+var (
+	DefaultSpotDaysBack = 30
```

Comment:
why change to var? Is it configurable?

Reply:
Yes, if using `ec2pricing` as a library, the caller can configure the pkg by modifying the var.